### PR TITLE
Turkish Translation

### PR DESCRIPTION
**Pull request by itudevops**
Hi everyone. As İstanbul Technical University IT Office, we have translated ui to Turkish.

## References
_Add references/links to any related issues or PRs. These may include:_
* Fixes #[issue-number]
* Requires DSpace/DSpace#[pr-number] (if a REST API PR is required to test this)

## Description
Short summary of changes (1-2 sentences).

## Instructions for Reviewers
Please add a more detailed description of the changes made by your PR. At a minimum, providing a bulleted list of changes in your PR is helpful to reviewers.

List of changes in this PR:
* First, ...
* Second, ...

**Include guidance for how to test or review your PR.** This may include: steps to reproduce a bug, screenshots or description of a new feature, or reasons behind specific changes. 

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_

- [ ] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & specs/tests), or I have provided reasons as to why that's not possible.
- [ ] My PR passes [TSLint](https://palantir.github.io/tslint/) validation using `yarn run lint`
- [ ] My PR doesn't introduce circular dependencies
- [ ] My PR includes [TypeDoc](https://typedoc.org/) comments for _all new (or modified) public methods and classes_. It also includes TypeDoc for large or complex private methods.
- [ ] My PR passes all specs/tests and includes new/updated specs or tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [ ] If my PR includes new, third-party dependencies (in `package.json`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.

### CODE REVIEW

**Comment by erpnedir** [comment]
I will be happy to contact ITUDEVOPS, discuss and review this PR.

**Comment by tdonohue** [comment]
@erpnedir : Please do feel free to review & provide your feedback.  I can only verify that the translation file "works" when I enable it.  But, I would appreciate the feedback of others on the actual translation.

File: src/assets/i18n/tr.json5 [hunk 1]
```diff
@@ -1,6748 +1,5126 @@
...
+  "menu.section.toggle.curation_task": "İyileştirme Görevi bölümünü aç/kapat",
   
   // "menu.section.toggle.edit": "Toggle Edit section",
-  // TODO New key - Add a translation
-  "menu.section.toggle.edit": "Toggle Edit section",
+  "menu.section.toggle.edit": "Düzenleme bölümünü aç/kapat",
   
   // "menu.section.toggle.export": "Toggle Export section",
-  // TODO New key - Add a translation
-  "menu.section.toggle
```
Comment (erpnedir) [inline src/assets/i18n/tr.json5:4133]: Item Öğe şeklinde çevrildiğinden burada Öge Türü daha doğru olmaz mı?

**Review by erpnedir — COMMENTED** [review]
Hi, firstly thank you very much for the translation. 
I propose to keep some of the Dspace main terminology for the community to better understand and adopt Dspace. Here are the terms I figured out. 
Community x Topluluk --> Komünite
Metadata x Meta veri --> Metadata
Bitstream x Veri Akışı --> Bitstream
Handle x Takma Ad --> Handle

**Comment by erpnedir** [comment]
Merhabalar,
ITÜ Bilgi İşlem Daire Başkanlığına çeviri için çok teşekkür ederiz.
Kurumun çeviride Türkçe'ye girmiş kelimeleri kullanma özenine saygı duyuyorum. Ancak naçizane bazı temel terimlerin olduğu gibi bırakılmasının anlaşılabilirlik açısından daha doğru olduğunu düşünüyorum. 
Örneğin, aşağıdaki terimler için, orjinal x ITU çevirisi --> Önerim
Community x Topluluk --> Komünite
Metadata x Meta veri --> Metadata
Bitstream x Veri Akışı --> Bitstream
Handle x Takma Ad --> Handle

**Comment by tdonohue** [comment]
@itudevops : Are you able to find time to make the small translations fixes suggested by @erpnedir ?  I was waiting on those small updates before moving this PR forward.  Thanks in advance though for your effort here!  I hope we can include this in 7.3.

**Comment by itudevops** [comment]
Hi all,

First of all, we thank @erpnedir for his recommendations. The translation was made by our part time crew. That's why, they are not so close to terminology. As a person in charge of Dspace, I agree with these fixes. In a short time, we will make fixes that are suggested.

**Comment by tdonohue** [comment]
@itudevops : That sounds good to me & thanks.  Once you have the chance to make the minor updates, I'll gladly move this PR forward & get it merged.  Just feel free to ping me (add a comment mentioning me) when the updates are made.

**Comment by tdonohue** [comment]
@itudevops : Just a quick reminder that this PR is only waiting on minor updates to the translation.  If you can find time to make them, I'll get this merged immediately.  Thanks!

**Comment by tdonohue** [comment]
@itudevops : It looks like this PR was forgotten about.  If you have a chance to make the minor updates, I'd appreciate it.  Otherwise, I may need to either reschedule this (for 7.4) or merge it as-is and find someone else to clean it up later.  Thanks!

**Comment by itudevops** [comment]
@tdonohue really sorry about this PR. We have some issues about our teams. That's why, it's been too late. In a couple hours, I will update translation file. May you please wait a little longer?

**Review by erpnedir — APPROVED** [review]
Düzenleme için çok teşekkürler. Metadata dışında diğerlerini yapmışsınız. Meta veri de bence kabul edilebilir.

**Comment by tdonohue** [comment]
@itudevops : I just tried to use this `tr.json5` translation, but unfortunately it won't load properly in DSpace 7 because the file appears to end abruptly (it looks to be cut off).  Most `json5` files have around 4,000-5,000 lines, but this `tr.json5` only has around 2,500 lines.

Could you see if maybe you forgot to push up some of the changes?  Are you able to get this `tr.json5` file to work properly in DSpace 7?

**Review by tdonohue — APPROVED** [review]
👍 Thanks @itudevops !  I tried this again today and it's working now after you fixed it up. 

I'll go ahead and merge this and follow it up with a small PR to enable it by default in the language dropdown.  Thanks again!

**Pull request merged**